website/source/docs/connect/discovery-chain.html.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: "docs"
+page_title: "Connect - Discovery Chain (beta)"
+sidebar_current: "docs-connect-discovery_chain"
+description: |-
+  A discovery chain is a series of distinct stages that a service discovery request flows through to discover and configure Connect proxy upstreams of the service type.
+---
+
+# Discovery Chain <sup>(beta)</sup>
+
+-> **Note:** This feature is not compatible with the 
+[built-in proxy](/docs/connect/proxies/built-in.html)
+or [native proxies](/docs/connect/native.html).
+
+A discovery chain is a series of distinct stages that a service discovery
+request flows through to discover and configure Connect proxy upstreams of the
```

Comment:
Unless the chain itself configures connect proxy upstreams this English is a bit weird. As I understand it the discovery chain is made of config entries. Config entries configure proxy upstreams.